packages/varnish/src/Footer.tsx

```diff
@@ -0,0 +1,51 @@
+import { Link } from '@mui/material';
+import * as React from 'react';
+
+import { Color, color as varnishColor } from './colors';
+
+export type LayoutVariant = 'left-aligned' | 'center-aligned';
+
+export type FooterVariant = 'default' | 'dark';
+
+interface Props {
+    variant?: FooterVariant;
+    children?: React.ReactNode | React.ReactNode[];
+    layout?: LayoutVariant; // TODO: Add layout understanding to new footer (Investigate Grid?)
+    backgroundColor?: Color;
+    className?: string;
+}
+
+export class Footer extends React.PureComponent<Props> {
+
+    render() {
+        const contrast = this.props.variant === 'dark';
+        // TODO: Update Contrast color palettes
+        const textColor = contrast ? varnishColor.white.toString() : varnishColor.black.toString();
+
+        // TODO: Add Link Theming and Typography (following 2 lines will be refactored after this TODO)
```

Comment:
please add a style update to the Link component to hold the colors we expect
see: https://mui.com/material-ui/customization/theme-components/
i.e., please add an override of the component `Link` to the varnish theme.

Reply:
ok to do in a separate PR since I'll need to investigate proper theming overrides? I have a TODO here for this, and created this issue: https://github.com/allenai/varnish-mui/issues/12